src/tests/ftest/interoperability/updown_grade.yaml

```
@@ -14,31 +18,32 @@ server_config:
         - D_LOG_FLUSH=DEBUG
 pool:
   control_method: dmg
-  scm_size: 1G
+  scm_size: 5G
   name: daos_server
 container:
   control_method: daos
   type: POSIX
-  properties: rd_fac:1
+  properties: rf:1
```

Should not be changed
```suggestion
  properties: rd_fac:1
```